<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="5" failed="8" ignored="0" total="23" passed="10">
  <reporter-output>
  </reporter-output>
  <suite name="Monopoly Test" duration-ms="56" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
    <groups>
    </groups>
    <test name="Class Tests" duration-ms="56" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
      <class name="test.TestCard">
        <test-method status="PASS" signature="setUpBeforeClass()[pri:0, instance:test.TestCard@7bfcd12c]" name="setUpBeforeClass" is-config="true" duration-ms="11" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeClass -->
        <test-method status="FAIL" signature="setUp()[pri:0, instance:test.TestCard@7bfcd12c]" name="setUp" is-config="true" duration-ms="2" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[\resources\scripts\go-to-jail.lua (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: \resources\scripts\go-to-jail.lua (The system cannot find the path specified)
at java.io.FileInputStream.open0(Native Method)
at java.io.FileInputStream.open(FileInputStream.java:195)
at java.io.FileInputStream.<init>(FileInputStream.java:138)
at core.Resources.stream(Unknown Source)
at core.Card.<init>(Unknown Source)
at test.TestCard.setUp(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="testGettersAndSetters()[pri:0, instance:test.TestCard@7bfcd12c]" name="testGettersAndSetters" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[\resources\scripts\go-to-jail.lua (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: \resources\scripts\go-to-jail.lua (The system cannot find the path specified)
at java.io.FileInputStream.open0(Native Method)
at java.io.FileInputStream.open(FileInputStream.java:195)
at java.io.FileInputStream.<init>(FileInputStream.java:138)
at core.Resources.stream(Unknown Source)
at core.Card.<init>(Unknown Source)
at test.TestCard.setUp(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGettersAndSetters -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:test.TestCard@7bfcd12c]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="tearDownAfterClass()[pri:0, instance:test.TestCard@7bfcd12c]" name="tearDownAfterClass" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterClass -->
      </class> <!-- test.TestCard -->
      <class name="test.TestStandardProperty">
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestStandardProperty@18e8568]" name="setUp" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="setUpBeforeClass()[pri:0, instance:test.TestStandardProperty@18e8568]" name="setUpBeforeClass" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeClass -->
        <test-method status="PASS" signature="testUpgradeAndDowngrade()[pri:0, instance:test.TestStandardProperty@18e8568]" name="testUpgradeAndDowngrade" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUpgradeAndDowngrade -->
        <test-method status="PASS" signature="testGettersAndSetters()[pri:0, instance:test.TestStandardProperty@18e8568]" name="testGettersAndSetters" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGettersAndSetters -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestStandardProperty@18e8568]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestStandardProperty@18e8568]" name="setUp" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestStandardProperty@18e8568]" name="setUp" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testRent()[pri:0, instance:test.TestStandardProperty@18e8568]" name="testRent" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testRent -->
        <test-method status="PASS" signature="tearDownAfterClass()[pri:0, instance:test.TestStandardProperty@18e8568]" name="tearDownAfterClass" is-config="true" duration-ms="1" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterClass -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestStandardProperty@18e8568]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestStandardProperty@18e8568]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- test.TestStandardProperty -->
      <class name="test.TestSprite">
        <test-method status="PASS" signature="setUpBeforeClass()[pri:0, instance:test.TestSprite@1c2c22f3]" name="setUpBeforeClass" is-config="true" duration-ms="1" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeClass -->
        <test-method status="FAIL" signature="testMiscSettersAndGetters()[pri:0, instance:test.TestSprite@1c2c22f3]" name="testMiscSettersAndGetters" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at core.Sprite.getWidth(Unknown Source)
at test.TestSprite.testMiscSettersAndGetters(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMiscSettersAndGetters -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestSprite@1c2c22f3]" name="setUp" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestSprite@1c2c22f3]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestSprite@1c2c22f3]" name="setUp" is-config="true" duration-ms="1" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testSpriteManipulation()[pri:0, instance:test.TestSprite@1c2c22f3]" name="testSpriteManipulation" duration-ms="3" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at core.ImageUtils.toBufferedImage(Unknown Source)
at core.ImageUtils.rotate(Unknown Source)
at core.Sprite.rotate(Unknown Source)
at test.TestSprite.testSpriteManipulation(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSpriteManipulation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestSprite@1c2c22f3]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="tearDownAfterClass()[pri:0, instance:test.TestSprite@1c2c22f3]" name="tearDownAfterClass" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterClass -->
      </class> <!-- test.TestSprite -->
      <class name="test.TestCardSpace">
        <test-method status="PASS" signature="setUpBeforeClass()[pri:0, instance:test.TestCardSpace@42f30e0a]" name="setUpBeforeClass" is-config="true" duration-ms="1" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeClass -->
        <test-method status="FAIL" signature="setUp()[pri:0, instance:test.TestCardSpace@42f30e0a]" name="setUp" is-config="true" duration-ms="1" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[\resources\config\chance.json (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: \resources\config\chance.json (The system cannot find the path specified)
at java.io.FileInputStream.open0(Native Method)
at java.io.FileInputStream.open(FileInputStream.java:195)
at java.io.FileInputStream.<init>(FileInputStream.java:138)
at core.Resources.stream(Unknown Source)
at core.CardSpace.parseConfig(Unknown Source)
at core.CardSpace.<init>(Unknown Source)
at test.TestCardSpace.setUp(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="setUp()[pri:0, instance:test.TestCardSpace@42f30e0a]" name="setUp" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="setUp()[pri:0, instance:test.TestCardSpace@42f30e0a]" name="setUp" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="testCardSprite()[pri:0, instance:test.TestCardSpace@42f30e0a]" name="testCardSprite" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[\resources\config\chance.json (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: \resources\config\chance.json (The system cannot find the path specified)
at java.io.FileInputStream.open0(Native Method)
at java.io.FileInputStream.open(FileInputStream.java:195)
at java.io.FileInputStream.<init>(FileInputStream.java:138)
at core.Resources.stream(Unknown Source)
at core.CardSpace.parseConfig(Unknown Source)
at core.CardSpace.<init>(Unknown Source)
at test.TestCardSpace.setUp(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCardSprite -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:test.TestCardSpace@42f30e0a]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:test.TestCardSpace@42f30e0a]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="testGetName()[pri:0, instance:test.TestCardSpace@42f30e0a]" name="testGetName" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[\resources\config\chance.json (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: \resources\config\chance.json (The system cannot find the path specified)
at java.io.FileInputStream.open0(Native Method)
at java.io.FileInputStream.open(FileInputStream.java:195)
at java.io.FileInputStream.<init>(FileInputStream.java:138)
at core.Resources.stream(Unknown Source)
at core.CardSpace.parseConfig(Unknown Source)
at core.CardSpace.<init>(Unknown Source)
at test.TestCardSpace.setUp(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetName -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:test.TestCardSpace@42f30e0a]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="setUp()[pri:0, instance:test.TestCardSpace@42f30e0a]" name="setUp" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:test.TestCardSpace@42f30e0a]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="SKIP" signature="testDrawCard()[pri:0, instance:test.TestCardSpace@42f30e0a]" name="testDrawCard" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[\resources\config\chance.json (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: \resources\config\chance.json (The system cannot find the path specified)
at java.io.FileInputStream.open0(Native Method)
at java.io.FileInputStream.open(FileInputStream.java:195)
at java.io.FileInputStream.<init>(FileInputStream.java:138)
at core.Resources.stream(Unknown Source)
at core.CardSpace.parseConfig(Unknown Source)
at core.CardSpace.<init>(Unknown Source)
at test.TestCardSpace.setUp(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDrawCard -->
        <test-method status="SKIP" signature="testCardCnt()[pri:0, instance:test.TestCardSpace@42f30e0a]" name="testCardCnt" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <exception class="java.io.FileNotFoundException">
            <message>
              <![CDATA[\resources\config\chance.json (The system cannot find the path specified)]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.io.FileNotFoundException: \resources\config\chance.json (The system cannot find the path specified)
at java.io.FileInputStream.open0(Native Method)
at java.io.FileInputStream.open(FileInputStream.java:195)
at java.io.FileInputStream.<init>(FileInputStream.java:138)
at core.Resources.stream(Unknown Source)
at core.CardSpace.parseConfig(Unknown Source)
at core.CardSpace.<init>(Unknown Source)
at test.TestCardSpace.setUp(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:523)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.io.FileNotFoundException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCardCnt -->
        <test-method status="SKIP" signature="tearDownAfterClass()[pri:0, instance:test.TestCardSpace@42f30e0a]" name="tearDownAfterClass" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterClass -->
      </class> <!-- test.TestCardSpace -->
      <class name="test.TestPlayer">
        <test-method status="PASS" signature="setUpBeforeClass()[pri:0, instance:test.TestPlayer@5b1d2887]" name="setUpBeforeClass" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeClass -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestPlayer@5b1d2887]" name="setUp" is-config="true" duration-ms="1" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testGiveMoney()[pri:0, instance:test.TestPlayer@5b1d2887]" name="testGiveMoney" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [1500] but found [1499]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [1500] but found [1499]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at test.TestPlayer.testGiveMoney(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGiveMoney -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestPlayer@5b1d2887]" name="tearDown" is-config="true" duration-ms="1" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestPlayer@5b1d2887]" name="setUp" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestPlayer@5b1d2887]" name="setUp" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testMiscSettersAndGetters()[pri:0, instance:test.TestPlayer@5b1d2887]" name="testMiscSettersAndGetters" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMiscSettersAndGetters -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestPlayer@5b1d2887]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="testProperties()[pri:0, instance:test.TestPlayer@5b1d2887]" name="testProperties" duration-ms="1" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testProperties -->
        <test-method status="FAIL" signature="testSetPlayerPos()[pri:0, instance:test.TestPlayer@5b1d2887]" name="testSetPlayerPos" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [39] but found [-1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [39] but found [-1]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at test.TestPlayer.testSetPlayerPos(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetPlayerPos -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestPlayer@5b1d2887]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestPlayer@5b1d2887]" name="setUp" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestPlayer@5b1d2887]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="FAIL" signature="testSetTurnsLeftInJail()[pri:0, instance:test.TestPlayer@5b1d2887]" name="testSetTurnsLeftInJail" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [1] but found [-1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [1] but found [-1]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at test.TestPlayer.testSetTurnsLeftInJail(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetTurnsLeftInJail -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestPlayer@5b1d2887]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestPlayer@5b1d2887]" name="setUp" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testTakeMoney()[pri:0, instance:test.TestPlayer@5b1d2887]" name="testTakeMoney" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [500] but found [501]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [500] but found [501]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at test.TestPlayer.testTakeMoney(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTakeMoney -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestPlayer@5b1d2887]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestPlayer@5b1d2887]" name="setUp" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testTurnsLeftInJail()[pri:0, instance:test.TestPlayer@5b1d2887]" name="testTurnsLeftInJail" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [5] but found [-1]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [5] but found [-1]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:652)
at org.testng.Assert.assertEquals(Assert.java:662)
at test.TestPlayer.testTurnsLeftInJail(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTurnsLeftInJail -->
        <test-method status="PASS" signature="tearDownAfterClass()[pri:0, instance:test.TestPlayer@5b1d2887]" name="tearDownAfterClass" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterClass -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestPlayer@5b1d2887]" name="setUp" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestPlayer@5b1d2887]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- test.TestPlayer -->
      <class name="test.TestUtilityProperty">
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestUtilityProperty@33e5ccce]" name="setUp" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testGetRent()[pri:0, instance:test.TestUtilityProperty@33e5ccce]" name="testGetRent" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetRent -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestUtilityProperty@33e5ccce]" name="tearDown" is-config="true" duration-ms="1" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUpBeforeClass()[pri:0, instance:test.TestUtilityProperty@33e5ccce]" name="setUpBeforeClass" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeClass -->
        <test-method status="PASS" signature="tearDownAfterClass()[pri:0, instance:test.TestUtilityProperty@33e5ccce]" name="tearDownAfterClass" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterClass -->
        <test-method status="PASS" signature="testGettersAndSetters()[pri:0, instance:test.TestUtilityProperty@33e5ccce]" name="testGettersAndSetters" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGettersAndSetters -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestUtilityProperty@33e5ccce]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestUtilityProperty@33e5ccce]" name="setUp" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
      </class> <!-- test.TestUtilityProperty -->
      <class name="test.TestDie">
        <test-method status="PASS" signature="setUpBeforeClass()[pri:0, instance:test.TestDie@24273305]" name="setUpBeforeClass" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeClass -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestDie@24273305]" name="setUp" is-config="true" duration-ms="2" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testGettersAndSetters()[pri:0, instance:test.TestDie@24273305]" name="testGettersAndSetters" duration-ms="1" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at core.Sprite.getWidth(Unknown Source)
at core.Die.getWidth(Unknown Source)
at test.TestDie.testGettersAndSetters(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGettersAndSetters -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestDie@24273305]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestDie@24273305]" name="setUp" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="testSetNum()[pri:0, instance:test.TestDie@24273305]" name="testSetNum" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSetNum -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestDie@24273305]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="tearDownAfterClass()[pri:0, instance:test.TestDie@24273305]" name="tearDownAfterClass" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterClass -->
      </class> <!-- test.TestDie -->
      <class name="test.TestRailroadProperty">
        <test-method status="PASS" signature="setUpBeforeClass()[pri:0, instance:test.TestRailroadProperty@46f5f779]" name="setUpBeforeClass" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeClass -->
        <test-method status="PASS" signature="testGetRent()[pri:0, instance:test.TestRailroadProperty@46f5f779]" name="testGetRent" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetRent -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestRailroadProperty@46f5f779]" name="setUp" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:test.TestRailroadProperty@46f5f779]" name="setUp" is-config="true" duration-ms="1" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestRailroadProperty@46f5f779]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="testGettersAndSetters()[pri:0, instance:test.TestRailroadProperty@46f5f779]" name="testGettersAndSetters" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGettersAndSetters -->
        <test-method status="PASS" signature="tearDownAfterClass()[pri:0, instance:test.TestRailroadProperty@46f5f779]" name="tearDownAfterClass" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownAfterClass -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:test.TestRailroadProperty@46f5f779]" name="tearDown" is-config="true" duration-ms="0" started-at="2018-04-18T22:21:13Z" finished-at="2018-04-18T22:21:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- test.TestRailroadProperty -->
    </test> <!-- Class Tests -->
  </suite> <!-- Monopoly Test -->
</testng-results>
